test/Tester/RelationalUtilities/PostgreSqlStorageForTesting.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using Orleans.SqlUtils;
+using UnitTests.General;
+
+namespace Tester.RelationalUtilities
+{
+    class PostgreSqlStorageForTesting : RelationalStorageForTesting
+    {
+        public PostgreSqlStorageForTesting(string connectionString)
+            : base(AdoNetInvariants.InvariantNamePostgreSql, connectionString)
+        {
+        }
+
+        public override string DefaultConnectionString
+        {
+            get { return @"Server=127.0.0.1;Port=5432;Database=postgres;Integrated Security=true;Pooling=false;"; }
```

Comment:
@veikkoeeva, I believe I have this set to the most reliable way now -- I kept pooling off and restored the code that kills sessions when dropping the database.